OracleWebCenterPortal/dockerfiles/README.md

```diff
@@ -0,0 +1,123 @@
+# Oracle WebCenter Portal 12.2.1.4.0 on Docker
+
+
+# 1. Introduction
+This project offers scripts to build an Oracle WebCenter Portal docker image based on 12.2.1.4.0 release. Use this Docker configuration to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Portal, see the [Oracle WebCenter Portal 12.2.1.4.0 Online Documentation](https://docs.oracle.com/en/middleware/webcenter/portal/12.2.1.4/index.html).
+
+This project creates Oracle WebCenter Portal Docker image with a single node targeted for development and testing.
+
+# 2. Hardware and Software Requirements
+Oracle WebCenter Portal has been tested and is known to run on the following hardware and software:
+
+## 2.1 Hardware Requirements
+
+| Hardware  | Size  |
+| :-------: | :---: |
+| RAM       | 16GB  |
+| Disk Space| 200GB+|
+
+## 2.2 Software Requirements
+
+|       | Version                        | Command to verify version |
+| :---: | :----------------------------: | :-----------------------: |
+| OS    | Oracle Linux 7.3 or higher     | more /etc/oracle-release  |
+| Docker| Docker version 17.03 or higher | docker version           |
```

Comment:
If you're using Oracle Linux (which is great), then you should be referring to [Oracle Container Runtime for Docker](https://docs.oracle.com/en/operating-systems/oracle-linux/docker/).